### ISSUE TITLE
Lázadó kormányzó

### ISSUE THREAD

**Issue by Pistapapa** [akarnokd/open-ig#1077]
Voltam olyan szerencsétlen, hogy elvesztettem a Zeuson bolygót egy Garthog támadás miatt majd jött az üzenet a lázadó kormányzóról és mikor menekülni akar, akkor ugye annak ellenére is a Zeuson bolygóról menekül, hogy már nem az enyém. Nem lehet ezt úgy megcsinálni, hogy ilyen esetben egy másik bolygó kormányzója az áruló ami még a birtokomban van? :)

**Comment by akarnokd** [comment]
Nem. A sztori és tartalom előre adott; nincs meg az eszköztárunk és tudásunk, hogy betömjük a lyukakat.
